osquery/killswitch/tests/killswitch_tests.cpp

```diff
@@ -0,0 +1,133 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
```

Comment:
nit: Looks like the copyright got a little messed up here, mind restructuring it?